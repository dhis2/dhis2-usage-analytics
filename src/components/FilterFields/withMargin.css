.uaa-fieldwrap {
    margin-bottom: 16px;
}
