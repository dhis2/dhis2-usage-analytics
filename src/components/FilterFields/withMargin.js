import React from 'react'
import './withMargin.css'

export default function withMargin(ComponentToWrap) {
    return function ComponentWithMargin(props) {
        console.log(props)
        return (
            <div className="uaa-fieldwrap">
                <ComponentToWrap {...props} />
            </div>
        )
    }
}
