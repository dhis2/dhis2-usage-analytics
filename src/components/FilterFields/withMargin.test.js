import React from 'react'
import { shallow } from 'enzyme'
import withMargin from './withMargin'

describe('withMargin', () => {
    const Component = props => <div className="wrapped" {...props} />
    const ComponentWithMargin = withMargin(Component)
    const tree = shallow(<ComponentWithMargin />)

    it('returns a function', () => {
        expect(typeof ComponentWithMargin).toEqual('function')
    })

    it('matches the snapshot', () => {
        expect(tree).toMatchSnapshot()
    })
})
