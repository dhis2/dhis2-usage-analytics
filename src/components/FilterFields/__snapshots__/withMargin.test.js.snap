// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withMargin matches the snapshot 1`] = `
<div
  className="uaa-fieldwrap"
>
  <Component />
</div>
`;
